README.md

```diff
@@ -38,11 +38,53 @@ main = runTwilio' (getEnv "ACCOUNT_SID")
   liftIO $ print calls
 
   -- Send a Message.
-  let body = PostMessage "+14158059869" "+14158059869" "Oh, hai"
+  let receivingPhone = "+14158059869"
+  let sendingPhone = "+14158059869"
+  let body = PostMessage receivingPhone sendingPhone "Oh, hai"
```

Comment:
Yep, works for me. I'll make the change and update the pull request tomorrow morning. 
-- 
Aaron